src/CanCommunity.ts

```diff
@@ -73,25 +82,32 @@ export class CanCommunity {
    */
   async execCode(
```

Comment:
@danielAlvess remembers to update the document too. I think you need to update markdown file.

Reply:
updated